src/common/utils/shuffle.util.ts

```diff
@@ -0,0 +1,17 @@
+// Utility function to shuffle an array (Fisher-Yates algorithm)
+export function shuffleArray(array) {
+  let currentIndex = array.length,
+    randomIndex;
+
+  while (currentIndex !== 0) {
+    randomIndex = Math.floor(Math.random() * currentIndex);
+    currentIndex--;
+
+    [array[currentIndex], array[randomIndex]] = [
+      array[randomIndex],
+      array[currentIndex],
+    ];
+  }
+
+  return array;
+}
```

Comment:
A remplacer par la method shuffle de [loadash](https://www.npmjs.com/package/lodash) ou [ramda](https://www.npmjs.com/package/ramda?activeTab=readme):

```typescript
import shuffle from 'lodash/shuffle';

export function shuffleArray(array) {
  return shuffle(array);
}
```

ou

```typescript
import { shuffle } from 'ramda';

export function shuffleArray<T>(array: T[]): T[] {
  return shuffle(array);
}
```

Tu peux delete la method et le fichier shuffle.utils.ts (les exemples ci-dessus c'est pour le context).

Reply:
leaving this as it is for now, i think it is not necessary to replace it for the moment.